### ISSUE TITLE
Click between 2 input fields; keyboard toggles on 1

### ISSUE THREAD

**Issue by kanetik** [Mottie/Keyboard#86]
This is an odd one, and I think it's new - but it might be related to other recent bugs.  I have 2 fields wired up to use the on-screen keyboard, as below:

$("input:text, textarea")
    .not("[readonly]")
    .keyboard({
        autoAccept: true,
    usePreview: false,
    maxLength: limit,
    layout: "custom",
    customLayout: {
        'default': layout,
        'shift': shiftLayout
    },
    display: {
        "close": "Close",
        "dotcom": ".com",
        "submit": submitText
    },
    position: position,
    visible: function (e, el) {
        $(".ui-keyboard-actionkey.ui-keyboard-close").addClass($(this).getkeyboard().options.css.buttonAction).end();
        $(".ui-keyboard-actionkey.ui-keyboard-submit").addClass($(this).getkeyboard().options.css.buttonAction).end();
    }
    });

All seems well when I click in the top field, then the second field, but when I click back into the first field, the keyboard closes.  Now, if I click back and forth between the fields, the top field alternates between showing the keyboard and hiding the keyboard, while the bottom field shows the keyboard on every click into it.

In other words, if I click between top and bottom fields, starting with top, this is what the keyboard does:

Click into top:        open
Click into bottom: open
Click into top:        close
Click into bottom: open
Click into top:        open
Click into bottom: open
Click into top:        close
Click into bottom: open

What I did notice is that on first displaying, the classes on the fields are slightly different:

Top: <input type="text" class="ui-keyboard-input ui-widget-content ui-corner-all ui-keyboard-autoaccepted" aria-haspopup="true" role="textbox">

Bottom: <input type="text" class="ui-keyboard-input ui-widget-content ui-corner-all" aria-haspopup="true" role="textbox">
